util/chplenv/utils.py

```diff
@@ -23,6 +23,13 @@ def get_chpl_home():
     return chpl_home
 
 @memoize
+def using_chapel_module():
```

Comment:
Comparisons to None in python should use `... is None` or `... is not None`. See the second point here: https://www.python.org/dev/peps/pep-0008/#programming-recommendations